README.md

```diff
@@ -28,7 +28,7 @@ A bundle will be produced from the native packager settings of this project. A b
 packager distribution and includes some component configuration. To load the bundle:
 
 ```bash
-loadBundle <HIT THE TAB KEY AND THEN RETURN>
+conductr:loadBundle <HIT THE TAB KEY AND THEN RETURN>
```

Comment:
As a scala dev I'd expect `namespace:...` to include both tasks and other keys I can `inspect` on, not only the DSL as seen in command line.

CLI-like commands would be an addition, these could use `conduct`, while keys are all in `conductr`.